docs/en-us/user/references/qos.md

```diff
@@ -0,0 +1,151 @@
+# Telnet (new version) Command Usage 
+dubbo 2.5.8 version reconstructs the telnet module, providing new telnet command support. 
+
+### Port
+the port of new version telnet is different from the port of dubbo protocol. The default port is `22222`, which can be changed by modifying configuration file `dubbo.properties`
+
+```
+dubbo.application.qos.port=33333
+```
+
+or by modifying the JVM parameter
+
+```
+-Ddubbo.application.qos.port=33333
+```
+
+### Safety
+
+In default situation, dubbo can receive any command sent from the host, which can be changed by modifying the configuration file `dubbo.properties`  
+
+```
+dubbo.application.qos.accept.foreign.ip=false
+```
+
+or by configuring the JVM parameter
+
+```
+-Ddubbo.application.qos.accept.foreign.ip=false
+```
+
+to reject command sent from the remote host, allowing only the local server to run the command 
+
+### Telnet and http protocol 
+
+The telnet module supports both http and telnet protocol for usage convenience under most situations. 
```

Comment:
maybe it is for why there are two protocol. 
 
so you can use like 'The telnet module supports both http and telnet protocol, in order to facilitate the use of various situations'